docs/templates/map-change-evidence.md

```
@@ -0,0 +1,106 @@
+### Documentation
+- [ ] README links verified (if applicable)
+- [ ] Map-related docs updated (if applicable)
+- [ ] AGENTS.md regenerated (if structure changed)
+
+---
+
+## Remaining Risks
+
+| Risk | Mitigation | Status |
+|------|------------|--------|
+| <!-- Describe any remaining risks --> | <!-- How it will be addressed --> | [ ] Resolved | [ ] Deferred |
```

markdownlint-cli2 (0.20.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)